src/utils/slack_notification.py

```diff
@@ -0,0 +1,57 @@
+import json
+import requests
+from config_key import SLACK_WEBHOOKS
+
+def GetTags(data):
+	'''
+	Generate array of all the tags listed in the newly registered API
+	'''
+	tags = []
+	if('tags' in data):
+		for item in data['tags']:
+			tags.append(item['name'])
+	return tags
+
+def GenerateSlackParams(data, res, user):
+	'''
+	Generate parameters that will be used in slack post request. 
+	In this case, markdown is used to generate formatting that 
+	will show in Slack message
+	'''
+	API_title = data["info"]["title"]
+	API_Description = data["info"]["description"]
+	API_id = res["_id"]
+	API_URL =  "http://smart-api.info/registry?q=" + API_id
+	block_markdown = "A new API has been registered on smartAPI.info:  \n\n\t *API Title:* " + API_title + "\n\n\t*API Description:* " + API_Description + "\n\n\t*SmartAPI Registry URL:* " + API_URL + "\n\n\t*By User:* " + user
```

Comment:
- smartAPI.info --> **S**martAPI.info
- API title and description should be capped for long text (using '...' at the end if it's too long).
- `\n\n` as new line gives extra line-space, maybe `<br />` is better? Alternatively, you can [string formating](https://realpython.com/python-f-strings/#f-strings-a-new-and-improved-way-to-format-strings-in-python) with multi-line string with `"""` here.
- this template can be the default template. As an enhanced feature, we can allow a customized template too (e.g. in the case of special tag(s), some text can be changed) from `SLACK_WEBHOOKS` setting